### PR TITLE
Cn ip addressing

### PR DESCRIPTION
**Pull request by Vaibhavpratapsingh22**
Chapter - 8 created 👍 
I enjoyed making the content of the chapter very interesting topics.

### CODE REVIEW

File: 6-computer-networks/8-ip-addressing.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    int fa0/4
+    switchport mode access
+    switchport access vlan 3
+    exit  [this will intialize the computer present at port 4 in VLAN 3]
+Now if you try to ping computer in VLAN B bya computer in VLAN A there will be connection time out message will appear which means we have successfully configured the VLAN setup.\
+This is the result when I tried to ping a Team B computer from team A computer.
+<p text align="center"><img src="https://user-images.githubusercontent.com/54719422/93597811-4a6b8000-f9d9-11ea-8da0-74ab0a974889.png"></p>
+
+This is the result when I tried to ping a Team A computer from Team B computer.
+<p text align="center"><img src="https://user-images.githubusercontent.com/54719422/93597818-4d667080-f9d9-11ea-8a3a-0ac28832a22a.png"></p>
```
Comment (udeysingh95) [inline 6-computer-networks/8-ip-addressing.md:144]: I would like to see the network as well in a screen shot.

**Review by udeysingh95 — CHANGES_REQUESTED** [review]
README is good updated. 
Some improvements: 
- [x] Add network images as well 
- [x] Line limit has crossed.

**Comment by Vaibhavpratapsingh22** [comment]
Screenshot (105)

**Comment by udeysingh95** [comment]
Network image is still missing. 
Screenshot of packet tracer 
Showing the setup of the network. You can also make vlans in it using red circles 
And mentioning their names as labels

**Comment by Vaibhavpratapsingh22** [comment]
OK @udeysingh95, I will add the image in the md file.
Yes, we can make multiple VLAN's using a single switch.

**Review by udeysingh95 — APPROVED** [review]

**Pull request merged**